### PR TITLE
integrated monsterapi to haystack

### PR DESCRIPTION
**Pull request by rishh76**
Monster API LLM models into Haystack.

### CODE REVIEW

File: monsterapi-integration.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+- **MonsterChatGeneraor**: Enables the use of MonsterAPI's language models within a Haystack pipeline.
+
+### Use MonsterAPI with Haystack
+
+Here's an example of how to use the `MonsterChatGeneraor` integration:
+
+```python
+from haystack import Pipeline
+from haystack.components.generators.chat import MonsterChatGeneraor
```
Comment (Vikasqblocks) [inline monsterapi-integration.md:48]: TYPO:

`from haystack.components.generators.chat import MonsterChatGenerator`

File: monsterapi-integration.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+from haystack.dataclasses import ChatMessage
+
+# Create chat messages
+messages = [ChatMessage.from_user("What's Natural Language Processing?")]
+
+# Initialize the MonsterChatGeneraor with your API key
+client = MonsterChatGeneraor(api_key="YOUR_MONSTER_API_KEY")
```
Comment (Vikasqblocks) [inline monsterapi-integration.md:55]: TYPO: 
`MonsterChatGenerator`

File: monsterapi-integration.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+layout: integration
+name: Monster API
+description: Use MonsterAPI with HeyStack
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
```
Comment (Vikasqblocks) [inline monsterapi-integration.md:10]: Here it should be monsterapi

File: monsterapi-integration.md (hunk 1 above)
Comment (Vikasqblocks) [reply monsterapi-integration.md]: Resolved!

File: monsterapi-integration.md (hunk 3 above)
Comment (Vikasqblocks) [reply monsterapi-integration.md]: Resolved!

File: monsterapi-integration.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+layout: integration
+name: Monster API
+description: Use MonsterAPI with HeyStack
```
[the remark this answers is not in the document]
Comment (Vikasqblocks) [reply monsterapi-integration.md]: Resolved!

**Comment by rishh76** [comment]
> Hello!
> 
> I also had a look at [deepset-ai/haystack-core-integrations#888](https://github.com/deepset-ai/haystack-core-integrations/pull/888) and I am inclined to close it to reduce maintenance efforts, since MonsterAPI can be used in Haystack via OpenAI Generators.
> 
> I suggest instead to modify the current PR showing how to use MonsterAPI in Haystack via OpenAI Generators. Similar examples:
> 
> * https://github.com/deepset-ai/haystack-integrations/blob/main/integrations/vllm.md
> * https://github.com/deepset-ai/haystack-integrations/blob/main/integrations/llamafile.md

Hello!

Make sense! Closed the core-integration PR and updated this PR to use OpenAIChatGenerator. 

Thank You.

**Comment by anakin87** [comment]
@rishh76 the best example to take inspiration from is Groq: https://github.com/deepset-ai/haystack-integrations/blob/main/integrations/groq.md

Please update this PR, checking that the code is correct and the text makes sense.

**Comment by anakin87** [comment]
@rishh76 looks much better.

Let's ask @bilgeyucel to have a look...

File: integrations/monsterapi.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
+---
+layout: integration
+name: MonsterAPI
+description: Use open Language Models served by MonsterAPI
+authors:
+    - name: monsterapi
+      socials:
+        github: qblocks
+        twitter: monsterapi
+        linkedin: monsterapi
```
Comment (bilgeyucel) [inline integrations/monsterapi.md:10]: Can you provide the full URL for the LinkedIn account? e.g. for deepset it is: https://www.linkedin.com/company/deepset-ai

**Review by bilgeyucel — APPROVED** [review]
Just left one minor comment ✅

File: integrations/monsterapi.md (hunk 5 above)
Comment (rishh76) [reply integrations/monsterapi.md]: > Can you provide the full URL for the LinkedIn account? e.g. for deepset it is: https://www.linkedin.com/company/deepset-ai


Sure. As requested, we have changed the link for LinkedIn in the new commit.

**Review by anakin87 — APPROVED** [review]

**Pull request merged**